prijateli_tree/app/database.py

```diff
@@ -180,12 +199,12 @@ class Player(Base):
     name_hidden = Column(Boolean, nullable=False)
     game = relationship(
         "Game",
-        foreign_keys="[Player.user_id, Player.game_id]",
+        foreign_keys="[game_players.user_id, game_players.game_id]",
         back_populates="players",
     )
     answers = relationship(
         "GameAnswer",
-        foreign_keys="[Player.user_id, Player.game_id]",
+        foreign_keys="[game_players.user_id, game_players.game_id]",
```

Comment:
This SHOULD fix your issue, @FedericoDM. Once this is merged, update your branch and see if these modifications work for you.